projects/dev/src/app/group/group-content/group-content.component.ts

```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit, ElementRef, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-group-content',
+  templateUrl: './group-content.component.html',
+  styleUrls: ['./group-content.component.scss']
+})
+export class GroupContentComponent implements OnInit {
+
+  @Input() data;
+  @Input() empty;
+
+  @Output() expandWholeWidth = new EventEmitter<void>();
+
+  constructor(
+    private elementRef: ElementRef
+  ) { }
+
+  ngOnInit() {
+  }
+
+  onTabChange(e) {
+    const tabs = this.elementRef.nativeElement.querySelectorAll('.mat-tab-labels .mat-tab-label');
+    const activeTab = this.elementRef.nativeElement.querySelector('.mat-tab-labels .mat-tab-label.mat-tab-label-active');
+    tabs.forEach((tab) => {
+      tab.classList.remove('mat-tab-label-before-active');
+    });
+    
+    let i;
+
+    for (i = 0 ; i < tabs.length ; i++) {
+      if (tabs[i] === activeTab) {
+        break;
+      }
+    }
+
```

Comment:
As you have lodash installed, do it properly with lodash.
Something like `let i = _.findIndex(tabs, activeTab)` (to be tested)

Reply:
Oh, my bad.
Why did I do that? It can be done with pure javascript itself.

Reply:
@smadbe 
I cleaned the code.

Reply:
There are 5 places where there is the same kind of code, fix all of them. If they were not part of this PR, fix the other ones in a branch from master directly.